doc/make.jl

```diff
@@ -0,0 +1,34 @@
+using Documenter, Clustering
+
+makedocs(
+    source = "source",
+    format = Documenter.HTML(prettyurls=false),
+    sitename = "Clustering.jl",
+    modules = [Clustering],
+    pages = [
+        "Clustering.jl Docs" => "index.md",
+        "Algorithms" => [
+            "algorithms.md",
+            "init.md",
```

Comment:
Initialization used to be a level-1 heading. Why make it level-2, given that in the overview you present it as one of the three top-level distinctions?

Reply:
IMHO having initialization on level-1 is a bit confusing as most of the people rarely use it. I'd rather remove its mention from `index.md`

Reply:
As you like, I just noticed that the overview isn't consistent with the organization of sections.